code-challenges.test.js

```diff
@@ -67,3 +116,26 @@ const vowelTester3 = "challenges"
 
 
 // b) Create the function that makes the test pass.
+// Pseudo Code:
+// Create a function called firstVowel
+// Use the indexOf method (Didn't work)
```

Comment:
I like that you are adding in here some what worked and what didn't.  It looks like you were able to get indexOf to work, just in a different placement of the function.